lib/player.js

```diff
@@ -3130,6 +3130,19 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return {'start': 0, 'end': 0};
   }
 
+  /**
+   * Go to live in a live stream.
+   *
+   * @export
+   */
+  goToLive() {
+    if (this.isLive()) {
+      this.video_.currentTime = this.seekRange().end;
+    } else {
+      shaka.log.warning('goToLive is for live streams!');
```

Comment:
If you want this to show up in production, then revise this to "alwaysWarn".  warning() will be removed in production builds.

If you're happy with this only showing up in debug, then it's fine as-is.

Reply:
I don't have a preference, I do what you prefer...

Reply:
It's fine with me.  Just wanted to make sure you knew you had options.